src/runtime/virtcontainers/qemu_arch_base.go

```diff
@@ -949,6 +952,24 @@ func (q *qemuArchBase) qomGetSlot(qomPath string, qmpCh *qmpChannel) (types.PciS
 	return types.PciSlotFromInt(slotNum)
 }
 
+// build initdata device
+func (q *qemuArchBase) buildInitdataDevice(ctx context.Context, devices []govmmQemu.Device, InitdataImage string) []govmmQemu.Device {
```

Comment:
We need to make `InitdataImage` to `initdataImage` because its prototype in the interface is:

```
buildInitdataDevice(ctx context.Context, devices []govmmQemu.Device, initdataImage string) []govmmQemu.Device
```